### ISSUE TITLE
Clarification concerning CF_EM info tag

### ISSUE THREAD

**Issue by PubuduSaneth** [dariober/cnv_facets#17]
Dear Dario

Thank you for the very useful FACETS interface. However, I would like to have a clarification concerning the CF_EM info tag.

According to your [readme file](https://github.com/dariober/cnv_facets), 
- **CF_EM** = Cellular fraction; fraction of DNA associated with the aberrant genotype. 

However, according to [R FACETS package](https://github.com/mskcc/facets),
- **CF_EM** = [Cellular fraction (cf) is the fraction of tumor cells with that event i.e it is already corrected for tumor purity.](https://github.com/mskcc/facets/issues/45)

Therefore, since R FACETS package corrects cellular fraction for the tumor purity, 
- I would imagine that **CF_EM tag in the cnv_facets output VCF file** would be the **fraction of tumor DNA associated with the aberrant genotype** 

Please let me know if I have misunderstood your readme file.

**Comment by dariober** [comment]
Hi- Thanks for your question. The CF_EM tag comes from FACETS' function emcncf which returns, among other things, a dataframe (cncf) with columns:

>cncf: dataframe consisting of the columns of segmentation output as well as cellular fraction (cf), total (tcn) and lesser (lcn) copy number of each segment and their em counterpart (with .em suffix) 

this where the CF_EM tag comes from. Since I don't manipulate the output of facets, what applies to the facets package applies also to my wrapper, cnv_facets.R

If I correctly remember, I got the description of cf.em from the vignettes where it says (emphasis mine):

> Once the observed total number is obtained we obtain the allele specific copy numbers m and p and the cellular fraction φ using the logOR data. **The cellular fraction is associated with the aberrant genotype**. For clonal copy number alterations, φ equals tumor purity. For subclonal events, φ will be lower than the overall sample purity.

TL;DR Your understanding is probably correct.

I'm going to close the issue just now but feel free to reopen.

Best
Dario

**Comment by GeorgetteTanner** [comment]
Hi

After running some tests on simulated datasets it is apparent that the CF_EM value is not adjusted for purity, which fits with with the above info "For clonal copy number alterations, φ equals tumor purity." 

Therefore I believe the CF_EM value does need to be adjusted for purity in order to get the "somatic cellular fraction".
